addon/utils/dom.js

```diff
@@ -0,0 +1,25 @@
+export function querySelector(node, query) {
```

Comment:
We don't need this now right?

Reply:
I don't think so.

Reply:
Y'all are right, this is not needed any more. It was included because of the earlier implementation and FastBoot combination.